Dockerfile

```diff
@@ -8,6 +8,12 @@ EXPOSE 8707
 
 RUN apk add --no-cache curl
 
-ADD blackfire blackfire-agent /usr/bin/
+COPY blackfire blackfire-agent /usr/local/bin/
+
+# Don't run as root
+RUN adduser -D blackfire
+RUN chown -R blackfire:blackfire /usr/local/bin
```

Comment:
Why did you change the whole directory? There is no need for that
(Same comment for the next line)

Reply:
I think `chown` and `chmod` commands are not needed at all.

Reply:
I thought it would be good that the user who runs the main process owns the files and that the permissions are explicitly set, even if the executable bit is set in git.

But, your'e right. It's not really needed. I can remove it if desired.

Reply:
I'm NOT a security expert, but it think a security breach in the agent would allow the hacker to change the binary.

Also, I checked with Debian and Alpine, all binaries are owned by root (with some rare exceptions).

IMHO, we should do the same.

Reply:
Alright, done :+1: